src/main/java/dk/kb/present/View.java

```diff
@@ -128,9 +136,15 @@ public String apply(DsRecordDto record) {
 
         }
 
+
         for (DSTransformer transformer: this) {
             try {
-                content = transformer.apply(content, metadata);
+                if (transformer.getStylesheet() != null && transformer.getStylesheet().contains("schemaorg2solr.xsl")){
+                    metadata.putIfAbsent("schemaorgjson", content);
```

Comment:
Nope, sorry, no hacking allowed 👮. The behaviour should be generalized and specified in the properties for the workflow. Alternatively a subclass of `XSLTTransformer` could be created.